source/installation-guide/virtual-machine.rst

```diff
@@ -39,7 +39,7 @@ Wazuh provides a pre-built virtual machine image (OVA) that you can directly imp
     # systemctl start logstash
     # systemctl status kibana
 
-5. In order to connect to the Kibana web user interface, login with http://OVA_IP_ADRESS:5601 (where ``OVA_IP_ADDRESS`` is your system IP).
+5. In order to connect to the Kibana web user interface, login with http://OVA_IP_ADRESS:443 (where ``OVA_IP_ADDRESS`` is your system IP).
```

Comment:
Here we will need to indicate the address: https://OVA_IP_ADRESSS instead to use the port.